src/Core/Package.vala

```diff
@@ -297,6 +297,17 @@ public class AppCenterCore.Package : Object {
         }
     }
 
+    private string? _author_id = null;
+    public string? author_id {
+        get {
+            if (_author_id != null) {
+                return _author_id;
+            }
+
+            return component.get_developer ().get_id ();
```

Comment:
```suggestion

            _author_id = component.get_developer ().get_id ();
            return _author_id;
```

You didn't need to remove substitution to `_author_id`, otherwise we need to run `component.get_developer ().get_id ();` every access to the `author_id` property

Reply:
Fixed!